Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -177,6 +195,7 @@ void cleanupInPlace(RWMol &mol, const CleanupParameters &params) {
 
 void cleanupInPlace(std::vector<RWMol *> &mols, int numThreads,
                     const CleanupParameters &params) {
+  throwIfMolPtrListContainsDuplicates(mols);
```

Comment:
You could also  just eliminate the duplicates and work on the rest.

```
  std::set<RWMol*> unique(mols.begin(), mols.end());
  std::vector<RWMol*> work_mols(unique.begin(), unique.end())
   standardizeMultipleMolsInPlace(
       static_cast<void (*)(RWMol &, const CleanupParameters &)>(cleanupInPlace),
       work_mols, numThreads, params);
```

Reply:
That's an option, but I think it's an error in the input.
Going that route also would require even more work if/when we add option of returning a list of "success,fail" values from the various InPlace functions

Reply:
You could still use the set as a faster version of the N**2 comparisons.

```
 if (std::unordered_set<RWMol*>(mols.begin(), mols.end()).size() != mols.size()) {
  throw ...
}
```